BEACON-V2-draft4-Model/genomicVariations/defaultSchema.json

```diff
@@ -2,69 +2,383 @@
+          "examples": [
+            { "id": "MONDO:0000001", "label": "disease or disorder" },
+            { "id": "HP:0000118", "label": "phenotypic abnormality" }
+          ]
+        },
+        "effect": {
+          "description": "Ontology term for the phenotypic or clinical effect. ",
+          "$ref": "https://raw.githubusercontent.com/ga4gh-beacon/beacon-framework-v2/main/common/ontologyTerm.json",
+          "examples": [
+            {
+              "id": "MONDO:0003582",
+              "label": "hereditary breast ovarian cancer syndrome"
+            },
+            { "id": "HP:0000256", "label": "macrocephaly" }
+          ]
+        },
+        "clinicalRelevance": {
+          "description": "A value from the five-tiered classification from the American College of Medical Genetics (ACMG) designed to describe the likelihood that a genomic sequence variant is causative of an inherited disease. (NCIT:C168798).",
+          "type": "string",
+          "enum": [
+            "benign",
+            "likely benign",
+            "uncertain significance",
+            "likely pathogenic",
+            "pathogenic"
+          ],
+          "example": "pathogenic"
+        },
+        "evidenceType": {
+          "description": "Ontology term for the type of evidence supporting variant-disease association from the Evidence & Conclusion Ontology (ECO). ",
+          "$ref": "https://raw.githubusercontent.com/ga4gh-beacon/beacon-framework-v2/main/common/ontologyTerm.json",
+          "examples": [
+            { "id": "ECO:0000361", "label": "inferential evidence" },
+            { "id": "ECO:0000006", "label": "experimental evidence" }
+          ]
+        },
+        "annotatedWith": {
+          "$ref": "#/definitions/SoftwareTool"
+        }
+      },
+      "required": ["conditionId", "effect"]
+    },
+    "FreqInPops": {
+      "type": "object",
+      "properties": {
+        "source": {
+          "type": "string",
+          "description": "The study",
+          "examples": [
+            "The Genome Aggregation Database (gnomAD)",
+            "The European Genome-phenome Archive (EGA)"
+          ]
+        },
+        "sourceReference": {
+          "type": "string",
+          "description": "A reference to further documentation or details.",
+          "examples": [
+            "https://gnomad.broadinstitute.org/",
+            "https://ega-archive.org/"
+          ]
+        },
+        "version": {
+          "type": "string",
+          "description": "version of the source data.",
+          "examples": ["gnomAD v3.1.1"]
+        },
+        "population": {
+          "type": "string",
+          "description": "A name for the population. A population could an ethnic, geographical one or just the 'members`of a study.",
+          "examples": [
+            "East Asian",
+            "ICGC Chronic Lymphocytic Leukemia-ES",
+            "Men",
+            "Children"
+          ]
+        },
+        "AF": {
```

Comment:
Is there a reason for using `AF` instead of `alleleFrequency`?

Reply:
There was one... but I can't remember it now  ;-)
so it shouldn't be that important.